src/autogluon_dashboard/plotting/framework_boxplot.py

```diff
@@ -0,0 +1,40 @@
+from typing import Union
+
+import hvplot
+
+from .all_plots import Plot
+
+
+class FrameworkBoxPlot(Plot):
+    def __init__(
+        self,
+        plot_title: str,
+        dataset_to_plot: hvplot.Interactive,
+        plot_type: str = "hvplot",
+        x_axis: Union[str, list] = None,
```

Comment:
I might have missed this in your previous PRs, but would recommend do List[type of elements] instead

Reply:
Sure!